schema.json

```diff
@@ -696,4 +696,4 @@
   },
   "additionalProperties": false,
   "$schema": "http://json-schema.org/draft-07/schema#"
-}
+}
```

Comment:
```suggestion
}

```

Reply:
Good catch, the end blank line rule has been implemented by the exporter script.